easybuild/easyconfigs/g/git-annex/git-annex-10.20230802-GCCcore-12.2.0.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'MakeCp'
+
+name = 'git-annex'
+version = '10.20230802'
+
+homepage = 'https://git-annex.branchable.com'
+description = """git-annex allows managing large files with git, without storing the file contents in git. It can sync,
+backup, and archive your data, offline and online. Checksums and encryption keep your data safe and secure. Bring the
+power and distributed nature of git to bear on your large files with git-annex."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+dependencies = [
+    ('GHC', '9.2.2', '-x86_64', SYSTEM),
+    ('GCC', '12.2.0', '', SYSTEM),
+    ('Stack', '2.11.1', '-x86_64', SYSTEM),
+    ('git', '2.38.1', '-nodocs'),
+]
+
+sources = [{
+    'git_config': {'url': 'git://git-annex.branchable.com',
+                   'repo_name': '%(name)s',
+                   'tag': '%(version)s',
+                   'clone_into': '%(name)s-%(version)s',
+                   },
+    'filename': '%(name)s-%(version)s.tar.gz',
+}]
+
+checksums = ['5c4cd3c0a1d141a1b85936fee3fa52f7638774c2146b138aa9ae218d3da4b89d']
+
+prebuildopts = "stack setup && stack build && "
+buildopts = "make install-bins BUILDER=stack PREFIX=%(builddir)s"
```

`buildopts` comes after the `make` command, so you should not repeat `make` here
```suggestion
buildopts = "install-bins BUILDER=stack PREFIX=%(builddir)s"
```